docs/CHANGELOG.asciidoc

```diff
@@ -35,6 +36,288 @@ This section summarizes the changes in each release.
+* Adds type icons to SingleFieldSelect component {pull}56313[#56313]
+* Disables style forms when they are not applied due to other style settings {pull}55858[#55858]
+* Autocompletes for custom color palettes and custom icon palettes {pull}56446[#56446]
+* Allows simultaneous opening of multiple tooltips {pull}57226[#57226]
+* Adds Top term aggregation {pull}57875[#57875]
+* Direct Discover &quot;visualize&quot; to open Maps application {pull}58549[#58549]
+* Top term percentage field property {pull}59386[#59386]
+* Adds UI to disable style meta and get top categories from current features {pull}59707[#59707]
+* Adds draw control to create distance filter {pull}58163[#58163]
+* Blended layer that switches between documents and clusters {pull}57879[#57879]
+* Default ES document layer scaling type to clusters and show scaling UI in the create wizard {pull}60668[#60668]
+* Disables add layer button when flyout is open {pull}54932[#54932]
+* Supports categorical styling for numbers and dates {pull}57908[#57908]
+Metrics::
+* Setup commonly used time ranges in timepicker {pull}56701[#56701]
+* Custom Metrics for Inventory View {pull}58072[#58072]
+* Alerting for metrics explorer and inventory {pull}58779[#58779]
+* Creates a new menu for observability links {pull}54847[#54847]
+* Creates Metric Threshold Alert Type and Executor {pull}57606[#57606]
+Monitoring::
+* Supports shipping directly to the monitoring cluster {pull}57022[#57022]
+Platform::
+* Improves validation in truncate field formatter editor {pull}56521[#56521]
+* Variable support for interpreter {pull}54788[#54788]
+Reporting::
+* Handles page setup errors and capture the page, don't fail the job {pull}58683[#58683]
+* Allows reports to be deleted in Management &gt; Kibana &gt; Reporting {pull}60077[#60077]
+SIEM::
+* Overview: Recent cases widget {pull}60993[#60993]
+* Adds custom reputation link {pull}57814[#57814]
+* Exports timeline {pull}58368[#58368]
+* Rule activity monitoring {pull}60816[#60816]
+* Removes has manage api keys requirement {pull}62446[#62446]
+* Adds release notes link and updates one UI section {pull}60825[#60825]
+* Adds rule notifications {pull}59004[#59004]
+* Version 7.7 rule import {pull}61903[#61903]
+* Creates ML Rules {pull}58053[#58053]
+* Case workflow api schema {pull}51535[#51535]
+* Service Now {kib} Action {pull}53890[#53890]
+* API with io-ts validation {pull}59265[#59265]
+* Status / Batch update {pull}59856[#59856]
+* Imports timeline {pull}60880[#60880]
+* Bug/clean up phase I {pull}61354[#61354]
+* Cases clean up Phase II {pull}61750[#61750]
+* Modifies gap detection util to accept all dateMath formats {pull}56055[#56055]
+* Adds note markdown field to backend {pull}59796[#59796]
+* Adds rule markdown field to rule create, detail, and edit flows {pull}60108[#60108]
+* Adds rule markdown to timeline global notes {pull}61026[#61026]
+Security::
+* Uses links instead of click handlers when switching spaces {pull}57730[#57730]
```

Comment:
```suggestion
* Uses links instead of click handlers when switching spaces {pull}57730[#57730]
* Introduce a login selector screen when multiple auth providers are enabled {pull}53010[#53010]
```